packages/sdk/src/ParadymWalletSdk.tsx

```diff
@@ -0,0 +1,79 @@
+import { type FullAgent, type SetupAgentOptions, setupAgent } from './agent'
+import { ParadymWalletMustBeInitializedError } from './error'
+import { useCredentials } from './hooks/useCredentials'
+import { useCredentialsById } from './hooks/useCredentialsById'
+import { AgentProvider, useAgent } from './providers/AgentProvider'
+
+/**
+ *
+ * Options that will be used when initializing the Paradym Wallet SDK
+ *
+ * Make sure to instantiate the `ParadymWalletSdk` once and use the same instance within the application, via for example a `useContext`
+ *
+ */
+export type ParadymWalletSdkOptions = {
+  /**
+   *
+   * Options that setup the internal agent.
+   *
+   * You can define here whether you want to use `openid4vc`, `didcomm` or both
+   *
+   * @see `SetupAgentOptions` for the specific details
+   *
+   */
+  walletOptions: SetupAgentOptions
```

Comment:
I think we need to start being consistent from the start, whether we use wallet or agent. Because the interface using `SetupAgentOptions`, but the property is called WalletOptions. 

Also, is there a specific reason to make these nested as wallet options? To me specifiying openid4vc top-level makes sense for a ParadymWalletSdk.

Reply:
Yeah I was thinking about this for too long. I just was not sure if there would be more options later that are not defined as `walletOptions`. I think having it top-level makes sense for now.